include/mbgl/util/geojson.hpp

```diff
@@ -1,22 +0,0 @@
-#pragma once
-
-#include <memory>
-
-namespace mapbox {
-namespace geojsonvt {
-class GeoJSONVT;
-} // namespace geojsonvt
-} // namespace mapbox
-
-namespace mbgl {
-
-class GeoJSON {
```

Comment:
Let's keep this header file and keep using the `GeoJSON` symbol, but change the definition to:

```
namespace mbgl {

using GeoJSON = mapbox::geojson::geojson;

} // namespace mbgl
```

This follows a convention established by `mbgl::Geometry`, `mbgl::Feature`, etc. It keeps the naming conventions within the project more consistent, and allows for definition changes (like this one) to happen with minimal changes to downstream code.